sdks/go/pkg/beam/core/runtime/xlangx/expand.go

```diff
@@ -111,12 +112,29 @@ func expand(
 		ext.ExpansionAddr = config
 	}
 
+	// The external transforms that needs to specify the output coder
+	// in expansion request sends tagged input as xlang.SetOutputCoder.
+	var set bool
+	for tag := range edge.External.InputsMap {
```

Comment:
Why is the tag on the InputsMap, not the outputs?

Reply:
The tag on the output is not retained after the output from the python expansion service. The python expansion service returns a `map{None: OutputPCol` rather than `map{tag: OutputPCol`

Reply:
replaced it. done.